Add `engineinfo` to script schema (#3601)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where the **coverage-analyze** command was not parsing the logs correctly.
* The content graph will now include the **python_version** field that each script/integration uses.
* Updated the **update-release-notes** command message structure when is run with **--force** flag.

> * Fixed an issue where *engineinfo* was not in the script schema.

Reviewer: ```suggestion
* Added the **engineinfo** in to the script schema. <explain shortly the purpose of the field>
```

* Fixed an issue where **validate** falsly failed with error `DS108` on descriptions ending with a newline.
* The content graph will now include the **python_version** field that each script/integration uses.
* Updated the **update-release-notes** command message structure when is run with **--force** flag.

> * Added the **engineinfo** in to the script schema (specifies which engine the script should run from).

Reviewer: ```suggestion
* Added the **engineinfo** in to the script schema. This field specifies on which engine the script will run.
```
